aloha-h2o/pom.xml

```diff
@@ -0,0 +1,157 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.eharmony</groupId>
+        <artifactId>aloha</artifactId>
+        <version>3.0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>aloha-h2o</artifactId>
+    <packaging>jar</packaging>
+    <name>${project.artifactId}</name>
+    <description>Provides an Aloha model for H2o For more info, see http://h2o.ai</description>
+    <inceptionYear>2013</inceptionYear>
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <url>${site.url}/${project.artifactId}</url>
+
+    <distributionManagement>
+        <site>
+            <id>${site.deploy.id}</id>
+            <!-- Note that GitHub does not like having leading slash here -->
+            <url>${site.deploy.url}${project.artifactId}</url>
+        </site>
+    </distributionManagement>
+
+    <properties>
```

Comment:
This version seems a little old.  Do you think it's worthwhile to upgrade to 3.6.0.3 as outline [here](http://h2o.ai/download/).  Also, if I remember correctly H2O guarantees NO cross version compatibility right?